transformers/component_tests/triggers/bid_event_trigger_test.go

```diff
@@ -0,0 +1,493 @@
+		Expect(bidEvents).To(ConsistOf(expectedEvent))
+	})
+
+	Specify("inserting a tick event triggers a bid_event insertion", func() {
+		address := test_data.EthFlipAddress()
+		addressID, addressErr := shared.GetOrCreateAddress(address, db)
+		Expect(addressErr).NotTo(HaveOccurred())
+		logID := test_data.CreateTestLog(headerOne.Id, db).ID
+		tickModel := test_data.TickModel()
+		tickModel.ColumnValues[event.HeaderFK] = headerOne.Id
+		tickModel.ColumnValues[event.AddressFK] = addressID
+		tickModel.ColumnValues[event.LogFK] = logID
+		expectedEvent := expectedBidEventNullStrings(tickModel, "tick", address, headerOne.BlockNumber)
+
+		insertErr := event.PersistModels([]event.InsertionModel{tickModel}, db)
+		Expect(insertErr).NotTo(HaveOccurred())
+
+		var bidEvents []bidEvent
+		queryErr := db.Select(&bidEvents, `SELECT bid_id, contract_address, act, lot, bid_amount, block_height FROM maker.bid_event`)
+		Expect(queryErr).NotTo(HaveOccurred())
+		Expect(bidEvents).To(ConsistOf(expectedEvent))
+	})
+
+	Specify("inserting a deal event triggers a bid_event insertion", func() {
+		address := test_data.EthFlipAddress()
+		addressID, addressErr := shared.GetOrCreateAddress(address, db)
+		Expect(addressErr).NotTo(HaveOccurred())
+		logID := test_data.CreateTestLog(headerOne.Id, db).ID
+		dealModel := test_data.DealModel()
+		dealModel.ColumnValues[event.HeaderFK] = headerOne.Id
+		dealModel.ColumnValues[event.AddressFK] = addressID
+		dealModel.ColumnValues[event.LogFK] = logID
+		expectedEvent := expectedBidEventNullStrings(dealModel, "deal", address, headerOne.BlockNumber)
+
+		insertErr := event.PersistModels([]event.InsertionModel{dealModel}, db)
+		Expect(insertErr).NotTo(HaveOccurred())
+
+		var bidEvents []bidEvent
+		queryErr := db.Select(&bidEvents, `SELECT bid_id, contract_address, act, lot, bid_amount, block_height FROM maker.bid_event`)
+		Expect(queryErr).NotTo(HaveOccurred())
+		Expect(bidEvents).To(ConsistOf(expectedEvent))
+	})
+
+	Specify("inserting a yank event triggers a bid_event insertion", func() {
+		address := test_data.EthFlipAddress()
+		addressID, addressErr := shared.GetOrCreateAddress(address, db)
+		Expect(addressErr).NotTo(HaveOccurred())
+		logID := test_data.CreateTestLog(headerOne.Id, db).ID
+		yankModel := test_data.YankModel()
+		yankModel.ColumnValues[event.HeaderFK] = headerOne.Id
+		yankModel.ColumnValues[event.AddressFK] = addressID
+		yankModel.ColumnValues[event.LogFK] = logID
+		expectedEvent := expectedBidEventNullStrings(yankModel, "yank", address, headerOne.BlockNumber)
+
+		insertErr := event.PersistModels([]event.InsertionModel{yankModel}, db)
+		Expect(insertErr).NotTo(HaveOccurred())
+
+		var bidEvents []bidEvent
+		queryErr := db.Select(&bidEvents, `SELECT bid_id, contract_address, act, lot, bid_amount, block_height FROM maker.bid_event`)
+		Expect(queryErr).NotTo(HaveOccurred())
+		Expect(bidEvents).To(ConsistOf(expectedEvent))
+	})
+
+	Describe("inserting events after diffs", func() {
```

Comment:
Not sure how we'd phrase it, but I wonder if we could specify in this Describe block, and the next one ("when diffs are inserted after events") that these tests are specific to the flip contract?